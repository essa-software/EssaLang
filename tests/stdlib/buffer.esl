func main() {
    mut buf = Buffer::zeros(1024);
    print("buf.size={}\n", buf.size());
    print("buf[0]={}\n", buf.get(0)); // 0
    for (let i of 0..512) {
        buf.set(i, 21);
    }
    print("buf[0]={} buf[600]={}\n", buf.get(0), buf.get(600)); // 21 0
}
