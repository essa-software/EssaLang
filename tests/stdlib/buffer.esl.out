buf.size=1024
buf[0]=0
buf[0]=21 buf[600]=0
